lucee-java/lucee-core/src/lucee/commons/io/TemporaryStream.java

```diff
@@ -137,8 +137,8 @@ public int available() throws IOException {
 
 		@Override
 		public void close() throws IOException {
-			ts.persis.delete();
 			is.close();
+			ts.persis.delete();
```

Comment:
Files were not getting deleted after reading, that is because the file still had a handle via the `InputStream`

Reply:
Doh!